docs/usage/models.md

```diff
@@ -853,7 +853,7 @@ try:
 except ValidationError as e:
     print(e)
     """
-    2 validation errors for InnerT[int]
+    2 validation errors for OuterT[int]
```

Comment:
Are these bugs that are getting fixed?

Reply:
yes https://github.com/pydantic/pydantic-core/issues/425

Reply:
nice!